pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -801,6 +801,30 @@ func (k *Bootstrapper) GenerateToken(cc config.ClusterConfig) (string, error) {
 	return joinCmd, nil
 }
 
+// StopKubernetes stops existing kubernetes.
+func StopKubernetes(stop bool, cr cruntime.Manager, k8s config.KubernetesConfig, runner command.Runner) {
```

Comment:
I suggest we still return the error  and the caller should log the error with klog.Warn that only shows up in verbose logs

Reply:
Hi, I have updated this function. In this case the function does not return a value, and we are retaining the logging of the original error:

`klog.Warningf("stop kubelet: %v", err)`

Have added additional out.Infof to indicate the Kubernetes status a the end of the function as well.